javav2/example_code/ec2/src/main/java/com/example/ec2/DescribeAccount.java

```diff
@@ -21,29 +21,40 @@
  */
 public class DescribeAccount {
     public static void main(String[] args) {
-        Region region = Region.US_EAST_1;
-        Ec2Client ec2 = Ec2Client.builder()
-                .region(region)
-                .build();
+        Ec2AsyncClient ec2AsyncClient = Ec2AsyncClient.builder()
+            .region(Region.US_EAST_1)
+             .build();
 
-        describeEC2Account(ec2);
-        System.out.print("Done");
-        ec2.close();
+        try {
+            CompletableFuture<DescribeAccountAttributesResponse> future = describeEC2AccountAsync(ec2AsyncClient);
+            future.join(); // Get the value.
+            System.out.println("EC2 Account attributes described successfully.");
+        } catch (RuntimeException rte) {

```

Comment:
At least one specific error should be caught. https://github.com/awsdocs/aws-doc-sdk-examples/wiki/general-code-example-standards#guidelines-4

Reply:
e.g. Ec2Exception should be caught, and the specific error code handled.

Reply:
We have an a few examples of handlign specific exceptions in this scenario now.